code/__DEFINES/role_preferences.dm

```diff
@@ -16,6 +16,7 @@
 #define ROLE_MALF "Malf AI"
 #define ROLE_MONKEY "Monkey"
 #define ROLE_OPERATIVE "Operative"
+#define ROLE_CLOWNOP "Clown Operative"
```

Comment:
You shouldn't need this because the antag_flag = ROLE_CLOWN_OPERATIVE, just use that instead.